app/views/files/index.slim

```diff
@@ -19,6 +19,8 @@ ul
 					input type="hidden" name="_method" value="DELETE"
 					button type="submit" Delete
 
+				a href="#{file.route}?download=true" Download
+
```

Comment:
Well, you're not really "editing" a directory, you're viewing it. But... hmm.. certainly edit/view is the more normal. Ok, we'll keep the routes like this :)